Исправить ошибку линтера

```
backend/app/database.py:9:1: W391 blank line at end of file
backend/app/models/db_models.py:9:25: W291 trailing whitespace
backend/app/models/db_models.py:76:24: W292 no newline at end of file
backend/app/routers/crud.py:116:38: W291 trailing whitespace
backend/app/routers/crud.py:185:[12](https://github.com/moevm/mse1h2024-telegram/actions/runs/8688295103/job/23823615050#step:5:13)1: E501 line too long (142 > 120 characters)
backend/app/routers/crud.py:218:1: W391 blank line at end of file
```

--- a/backend/app/database.py
+++ b/backend/app/database.py
@@ -6,4 +6,3 @@ from .models.db_models import Table, Teacher, Log, TelegramUser, Statistic
 async def init_db(address: str, name: str):
     client = motor.motor_asyncio.AsyncIOMotorClient(address)
     await init_beanie(database=client[name], document_models=[Teacher, Log, Table, TelegramUser, Statistic])
-

--- a/backend/app/models/db_models.py
+++ b/backend/app/models/db_models.py
@@ -6,7 +6,7 @@ from pydantic import BaseModel
 from beanie.odm.fields import Indexed
 
 
-class Status(str, Enum): 
+class Status(str, Enum):
     SENDED = "SENDED"
     CONFIRMED = "CONFIRMED"
     ERROR = "ERROR"

--- a/backend/app/routers/crud.py
+++ b/backend/app/routers/crud.py
@@ -113,7 +113,7 @@ async def delete_table(table_id: PydanticObjectId):
 
 
 @router.delete("/users/{username}", response_model=TelegramUser)
-async def delete_user(username: str):                             
+async def delete_user(username: str):
     query = await TelegramUser.find(TelegramUser.username == username).to_list()
     if not query:
         raise HTTPException(
@@ -182,8 +182,8 @@ async def edit_user(user_id: PydanticObjectId, username: str, chat_id: str):
 
 
 @router.put("/table/{table_id}/{page_id}", response_model=Table)
-async def edit_rule(table_id: PydanticObjectId, page_id: str, new_name: str, t_col: str, column1: str, column2: str, comparison_operator: str,
-                    text: str):
+async def edit_rule(table_id: PydanticObjectId, page_id: str, new_name: str, t_col: str,
+                    column1: str, column2: str, comparison_operator: str, text: str):
     table = await Table.get(table_id)
     if not table:
         raise HTTPException(
@@ -215,4 +215,3 @@ async def edit_teacher(teacher_id: PydanticObjectId, telegram_login: str, names_
         )
     await teacher.set({Teacher.telegram_login: telegram_login, Teacher.names_list: names_list})
     return teacher
-

